Sources/NIOTransportServices/StateManagedNWConnectionChannel.swift

```diff
@@ -100,6 +104,8 @@ extension StateManagedNWConnectionChannel {
         self
     }
 
+    public var isDatagramChannel: Bool { false }
```

Comment:
This can avoid being `public`, as this field isn't part of a public protocol so its visibility won't actually be `public` anyway.